getting_started/first_3d_game/04.mob_scene.rst

```diff
@@ -198,12 +201,13 @@ We got a random position, now we need a ``random_speed``. ``randi_range()`` will
     {
         // ...
 
-        // We calculate a random speed.
-        float randomSpeed = (float)GD.RandRange(MinSpeed, MaxSpeed);
+        // We calculate a random speed (integer).
+        int randomSpeed = GD.RandRange(MinSpeed, MaxSpeed);
```

Comment:
Why an integer speed? Speed can definitely be continuous, I would change the Min/MaxSpeed definition to float instead, tho that would require changing all the other speeds to float as well.

Reply:
AFAIK, that's the type used in GDScript so that's what I used.